connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SourceTaskOffsetCommitter.java

```diff
@@ -63,7 +63,7 @@
     public SourceTaskOffsetCommitter(WorkerConfig config) {
         this(config, Executors.newSingleThreadScheduledExecutor(ThreadUtils.createThreadFactory(
                 SourceTaskOffsetCommitter.class.getSimpleName() + "-%d", false)),
-                new ConcurrentHashMap<ConnectorTaskId, ScheduledFuture<?>>());
+            new ConcurrentHashMap<>());
```

Comment:
nit: format got unaligned. Please check the suggestion fixes it
```suggestion
                new ConcurrentHashMap<>());
```